test-project/src/main/java/com/bsu/test_project/StatusServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.bsu.test_project;
+
+import java.io.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+
+@WebServlet(name = "statusServlet", value = "/status")
+public class StatusServlet extends HttpServlet {
+    private String message;
+
+    public void init() {
+        message = "The application is running";
```

Comment:
The idea is clear, but still better to declare this as a constant
`private static final String MESSAGE = "The application is running";`

Reply:
It's considered a good practice to declare whatever literals as constants. In a pro solution, the "<h1 style = 'color: red'" would be a constant as well. Not to say only, that in a pro solution the markup has usually nothing to do with the Java code)